Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -116,6 +127,14 @@ void UPSWorldSubsystem::RegisterSpotComponent(UPSSpotComponent* MySpotComponent)
 		return;
 	}
 	PSSpotComponentArrayInternal.AddUnique(MySpotComponent);
```

Comment:
You have just added TMap, please dont support `PSSpotComponentArrayInternal` array anymore: remove its property along with all usages like one in `GetCurrentSpot()`, where you can simply call `FindSpotByRow(CurrentRowNameInternal)`, which method was asked to introduce in previous comment

Reply:
done https://github.com/moinm3uw/ProgressionSystem/pull/59/commits/bea714bb5069518ff99eefdc5edba86001ecda6f